std/build.zig

```diff
@@ -1648,6 +1648,10 @@ pub const TestStep = struct {
         self.verbose = value;
     }
 
+    pub fn addIncludeDir(self: &TestStep, path: []const u8) void {
+        self.builder.include_paths.append(path) catch unreachable;
```

Comment:
This does fix the issue, but it adds the include directory globally. I think it should add the directory for only this build artifact.